docs/examples/warp.ipynb

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
Please add more narrative with the detailed description of what is happening here. Now it is not something we can call a tutorial.
You can also show how to achieve things that were not possible earlier, like Shear or Translate operator.
Also please add this notebook to our docs (update appropriate *.rst inside docs folder to include this file).